app/experimenter/experiments/tests/test_tasks.py

```diff
@@ -332,6 +332,62 @@ def test_experiment_with_no_creator_in_recipe(self):
             ).exists()
         )
 
+    def test_experiment_with_pause_true(self):
```

Comment:
This happens to me all the time, there's some polynomial combination of states and if you test every single one of them you end up with this huge battery of nearly identical tests.  Try to cut this down to two test cases.  Think about it from what cases the code cares about rather than the possible state combinations.

Reply:
Like Case 1 where state stays the same and Case 2 where it changes?

Reply:
BINGO